content/applications/general/in_app_purchase.rst

```diff
@@ -2,58 +2,142 @@
+
+   The prepaid credits for the SMS service are automatically deducted from the existing pool. If
+   there are not enough credits to send the message, Odoo will prompt the purchase of additional
+   credits.
+
+For more information on how to use each service, review the documentation below:
+
+.. seealso::
+   - :doc:`Lead mining </applications/sales/crm/acquire_leads/lead_mining>`
+   - :doc:`Enrich your contacts base with Partner Autocomplete
+     </applications/sales/crm/optimize/partner_autocomplete>`
+   - :doc:`SMS essentials </applications/marketing/sms_marketing/essentials/sms_essentials>`
+
+.. _in_app_purchase/credits:
+
+IAP credits
+===========
+
+Every time an IAP service is used, the prepaid credits for that service are spent. Odoo prompts the
+purchase of more credits when there are not enough credits to continue using the service. Email
+alerts can also be set up for when :ref:`credits are low <in_app_purchase/low-credits>`.
+
+Credits are purchased in "packs" from the `Odoo IAP website
+<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.
+
+.. example::
+   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available for
+   purchase, in denominations of:
+
+   - **10** credits
+   - **100** credits
+   - **500** credits
+   - **1,000** credits
+
+   The number of credits consumed depends on the length of the SMS and the country of destination.
+   For more information on SMS pricing, click :doc:`here
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>`.
 
-.. image:: in_app_purchase/image2.png
-   :align: center
+.. _iap/buying_credits:
 
-.. tip::
-   An IAP account can be disabled by appending `+disabled` to its token.
-   Reverting this change will re-enable the account.
+Buy credits
+-----------
+
+To manually buy credits:
 
-IAP Portal
-==========
+#. Open the :guilabel:`Settings app`.
+#. In the search bar, type :guilabel:`IAP`.
+#. Click :guilabel:`View My Services`.
+#. On the :guilabel:`IAP Account Settings` page, click on the service that needs more credits.
+#. On the :guilabel:`Account Information` page, click the :guilabel:`Buy Credit` button.
 
-The IAP Portal is a platform regrouping your IAP Services. It is accessible from
-:menuselection:`Settings app --> Odoo IAP --> View my Services`. From there, you can view your current
-balance, recharge your credits and set a reminder when your balance falls below a threshold.
+   - The balance of credits is displayed below the :guilabel:`Buy Credit` button.
+#. Click :guilabel:`Buy` on the desired pack of credits, and follow the prompts to enter payment
+   details.
+#. Once the transaction is complete, the credits will be available for use.
 
-.. image:: in_app_purchase/image3.png
+.. image:: in_app_purchase/account-info.png
    :align: center
+   :alt: The Account Information page for an IAP service showing the Buy Credit button.
```

Comment:
This would be nice to see an example IAP service with highlighted parts of the image. (IE red box around Buy credit)